doc/api/errors.md

```diff
@@ -719,6 +719,19 @@ An attempt was made to register something that is not a function as an
 The type of an asynchronous resource was invalid. Users are also able
 to define their own types if using the public embedder API.
 
+<a id="ERR_BLOB_OPEN_FAILED"></a>
+
+### `ERR_BLOB_OPEN_FAILED`
+
+Opening a file path as a `Blob` failed.
```

Comment:
I’d mark this errors as experimental since it should ideally just go away once the features leaves experimental state and match normal fs errors.